.github/workflows/snap.yaml

```diff
@@ -1,5 +1,11 @@
 name: snap CI
-on: ["pull_request"]
+on:
+  pull_request:
+    paths:
+      - "**/*.go"
+      - "**/*.rs"
```

Comment:
You probably want to add some or all of:

- `*.toml` for `Cargo.toml`
- `*.mod` for golang dep tracking
- `Makefile` and `*.mk`?
- `*.sh`?

Reply:
yeah - thanks, PR updated